hax/hax/handler.py

```diff
@@ -102,11 +117,13 @@ def pull_msg():
                         decorated = (repeat_if_fails(wait_seconds=5))(fn)
                         decorated(item)
                     elif isinstance(item, BroadcastHAStates):
-                        LOG.info('HA states: %s', item.states)
+                        LOG.info('HA states: %s Current statuses: %s',
```

Comment:
It looks like a debug logging to me. If it doesn't help in common maintenance activities, please remove it. Otherwise please use `LOG.debug` instead.

Reply:
Removed.